src/main/java/org/prebid/server/util/ObjectUtil.java

```diff
@@ -0,0 +1,25 @@
+package org.prebid.server.util;
+
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public final class ObjectUtil {
+
+    private ObjectUtil() {
+    }
```

Comment:
Let's use approach from 'Application' class. Add `@SuppressWarnings("checkstyle:hideutilityclassconstructor")` and remove constructor

Reply:
i'd rather to keep it simple and stay with private constructor.